test/rules/member-access/no-public/test.ts.lint

```diff
@@ -6,6 +6,9 @@ class {
     public/*some comment*/ get y() {}
     ~~~~~~ [0]
     public() {}
+    constructor(public readonly param : number) {}
```

Comment:
consider adding the `public` keyword to the `constructor` to make sure the rule finds both failures